src/ImageSharp.Drawing/DrawImage.cs

```diff
@@ -7,7 +7,7 @@ namespace ImageSharp
 {
     using System;
 
-    using Processors;
+    using Drawing.Processors;
```

Comment:
Nope... Processing is a library name, processors should be anything inheriting ImageProcessor<TColor>